src/common/util.ts

```diff
@@ -39,3 +42,26 @@ export const checkArgumentValid = (expression: any, code: number) => {
     throw new ArgumentError({ code });
   }
 };
+
+export const systemInfoString = () => {
+  return `NodeSDK/${version} (Node ${
+    process.version
+  }; ${os.platform()} ${os.release()} )`;
+};
+
+export const generateUserAgent = (
+  clients: ConnectorVersion[] | undefined,
```

Comment:
you can also use optional here: `clients?: ConnectorVersion[]` and `drivers?: ConnectorVersion[]`